src/json-symtab-language/json_symbol.h

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************\
+
+Module: JSON symbol deserialization
+
+Author: Chris Smowton, chris.smowton@diffblue.com
+
+\*******************************************************************/
+
+#ifndef CPROVER_UTIL_JSON_SYMBOL_H
+#define CPROVER_UTIL_JSON_SYMBOL_H
+
+#include "json.h"
```

Comment:
@NlightNFotis Shouldn't it be `#include <util/json.h>`? At least I can't compile it locally like this.

Reply:
yes, that's fixed already, sorry I hadn't pushed.